crates/rpc/rpc-types/src/eth/trace/geth/mod.rs

```diff
@@ -313,6 +313,58 @@ pub struct GethDefaultTracingOptions {
 }
 
 impl GethDefaultTracingOptions {
+    /// build a new GethDebugTracingOptions with default value
+    pub fn new() -> Self {
+        GethDefaultTracingOptions::default()
+    }
```

Comment:
new that calls default is redundant, it's always better to reserve new in case this will be needed eventually.

please remove.